util/check_dif_statuses.py

```diff
@@ -229,6 +245,30 @@ def _get_funcs(self, file_path):
                     funcs.add(result.group(1))
         return funcs
 
+    def _get_expected_irqs(self):
+        # TODO(timothytrippel): parse HJSON to get expected IRQ info per IP
+        logging.error(f"{self._get_expected_irqs.__name__}() unimplemented.")
+        sys.exit(1)
+
+    def _get_expected_alerts(self):
+        # TODO(timothytrippel): parse HJSON to get expected Alert info per IP
+        logging.error(f"{self._get_expected_irqs.__name__}() unimplemented.")
+        sys.exit(1)
+
+    def _get_irq_funcs_defined(self, defined_funcs):
+        irq_funcs = set()
```

Comment:
Probably add assertion here to ensure `defined_funcs` is defined, since this implicitly assumes that `_get_defined_funcs` has been called. Ditto below.

Reply:
Fixed.